docs/TRANSLATING.md

```diff
@@ -2,13 +2,29 @@
 
 This guide is written to provide instructions from scratch for contributing to the translation of the Jamulus application to other languages.
 
-The code for Jamulus is open source, and is managed and made available via the Github site.
+The code for Jamulus is open source, and is managed and made available via the Github site. However, translators are not required to use GitHub. Jamulus is available on [Hosted Weblate](https://hosted.weblate.org/projects/jamulus/).
 
-For completeness, this document describes the use both of GitHub (using `git`) and of _Qt Linguist_.
+You have two options to translate Jamulus:
+1. Using [Weblate](https://hosted.weblate.org/projects/jamulus/jamulus-app/) (online translation, easier and more feature rich. However, as of 09/2022, it will show your email address on GitHub.) (recommended)
+2. Using git directly (more advanced, but you can test your changes on your machine locally)
 
 ---
 
-## Introduction
+## Introduction (using Weblate)
+
+For easier communication, a translator should - if possible - a GitHub account as described in the **1. Setting up** paragraph in the git section below and use it to [create an account on Hosted Weblate](https://hosted.weblate.org/accounts/register/). Ensure that the Jamulus main developers are aware of your intentions to translate to your chosen language before beginning (e.g. by [creating or commenting on a respective issue during the release process for your language on GitHub](https://github.com/jamulussoftware/jamulus/issues)). This ensures you will have proper access and support provided. You can have a look at the [adding a new language (checklist)](#adding-a-new-language-checklist) to see what to expect.
+
+## Using Weblate for translation
+
+After having logged in to Weblate - best with a linked GitHub account - visit the [Jamulus project page on Weblate](https://hosted.weblate.org/projects/jamulus/). There are several components dedicated to the Jamulus project. If you want to translate e.g. the Jamulus application, click on "Jamulus app" and choose your language. After that, translate the strings displayed on the Website. Weblate will open or update a Pull Request on the Jamulus repo in the next hours, where your translation will be reviewed.
```

Comment:
```suggestion
visit the [Jamulus project page on Hosted Weblate](https://hosted.weblate.org/projects/jamulus/) after logging in there Weblate (preferably with a GitHub account). The Jamulus translation effort consists of multiple components. Click on "Jamulus app" and choose your language to translate e.g. the Jamulus application. After that, translate the strings displayed on the Website. Weblate will open or update a pull request in the Jamulus Git repository within the next few hours, where your translation will be reviewed.
```

Reply:
> Click on "Jamulus app" and choose your language to translate e.g. the Jamulus application.

No, clicking on "Jamulus app" _will_ let you choose the languge for the Jamulus application.  This is what this page is about.  There should be a mention somewhere of the installer, though.  Maybe both should be here.  None of the other components are relevant, though.

Reply:
@pljones I updated the comment.
They are all linked components though, so translating "Jamulus app" should suffice.
I don't understand "After that, translate the strings displayed on the Website."
How can I find those?

It is possible to adjust the ranking of WL components to put the important ones up top.

Reply:
If you're translating the app and installer, only the app and installer ones need touching.

If you're updating the website, my understanding is it's everything _except_ the app and installer ones.

It's like this because of how Weblate and where the source comes from interact, as far as I understand.

Reply:
@pljones is correct. I've nevertheless applied some changes.